jerry-core/jerry.c

```diff
@@ -1633,23 +1633,23 @@ jerry_init (jerry_flag_t flags) /**< combination of Jerry flags */
 
   if (flags & (JERRY_FLAG_MEM_STATS | JERRY_FLAG_MEM_STATS_SEPARATE))
```

Comment:
at some point the J should be added before these macros as well for consistency but the patch is big enough.